admin/views/hdf/review-pupil-details.ejs

```diff
@@ -45,7 +45,10 @@
                 <tr>
                     <td<% if (messages.urlSlug && messages.urlSlug == pupil.urlSlug) { %> class="highlight-item"<% } %>>
                         <div class="highlight-wrapper" id="anchor-<%= pupil.urlSlug %>">
-                            <span><%= pupil.lastName %>, <%= pupil.foreName %></span>
+                            <span><%=
+                              `${pupil.lastName}, ${pupil.foreName}` +
+                              (pupil.showMiddleNames ? ` ${pupil.middleNames}` : '')
```

Comment:
Something seems off here.  Usually, we use the `fullName` property provided by the `pupilIdentificationFlagService`.